### PR TITLE
Similar Items/Explore more items 

### PR DESCRIPTION
**Pull request by lbiedinger**

### CODE REVIEW

File: components/item/ItemPreviewCard.vue [hunk 1]
```diff
@@ -54,7 +54,7 @@
 
     computed: {
       texts() {
-        if (this.variant === 'similar') return [];
+        if (this.variant === 'similar' || this.variant === 'explore') return [];
```
Comment (lbiedinger) [inline components/item/ItemPreviewCard.vue:57]: I'd use a check to see if the variant is in an array, but similar will be removed after the new page becomes the default.

File: assets/scss/cards.scss [hunk 2]
```diff
@@ -113,6 +113,38 @@
...
+    flex: 100%;
+    margin-bottom: 1rem;
+    min-height: auto;
+    margin-left: 0px;
+    margin-right: 0px;
+
+    @media (min-width: $bp-medium) {
+      &:first-child {
+        margin-left: 0px;
+        margin-right: 8px;
```
Comment (alansutherland) [inline assets/scss/cards.scss:126]: switch to 0.5rem

File: assets/scss/cards.scss [hunk 3]
```diff
@@ -113,6 +113,38 @@
...
+      }
+
+      &:last-child {
+        margin-left: 8px;
```
Comment (alansutherland) [inline assets/scss/cards.scss:130]: switch to 0.5rem

File: assets/scss/cards.scss [hunk 4]
```diff
@@ -113,6 +113,38 @@
...
+        margin-right: 0px;
+      }
+    }
+
+    @media (min-width: $bp-large) {
+      &:first-child {
+        margin-right: 4px;
```
Comment (alansutherland) [inline assets/scss/cards.scss:137]: switch to 0.25rem

File: assets/scss/cards.scss [hunk 5]
```diff
@@ -113,6 +113,38 @@
...
+      }
+
+      &:last-child {
+        margin-left: 4px;
```
Comment (alansutherland) [inline assets/scss/cards.scss:141]: switch to 0.25rem for this and the two lines below

File: assets/scss/variables.scss [hunk 6]
```diff
@@ -48,6 +48,7 @@ $font-size-extrasmall: 0.75rem;
 $font-size-smallest: 0.625rem;
 
 $grid-gutter: 15px;
+$grid-gutter-narrow: 4px;
```
Comment (alansutherland) [inline assets/scss/variables.scss:51]: switch to 0.25rem

**Review by alansutherland — CHANGES_REQUESTED** [review]
The h2 title should have margin-bottom 0.5rem according to the design

Below the cards there’s a 112px gap before the footer on the design, currently it looks around 40px. Perhaps add this as padding-bottom to .container wrapping the title, metadata and cards? that way the gap will also be on pages without similar items.

**Review by alansutherland — APPROVED** [review]

**Review by mirjamverloop — APPROVED** [review]

**Review by mirjamverloop — APPROVED** [review]

**Pull request merged**